pkg/cmd/network/inspect.go

```diff
@@ -58,25 +58,39 @@ func Inspect(ctx context.Context, client *containerd.Client, options types.Netwo
 		}
 
 		network := netList[0]
-		var filters = []string{fmt.Sprintf("labels.%q==%q", labels.Networks, []string{network.Name})}
-
-		filteredContainers, err := client.Containers(ctx, filters...)
 
+		allContainers, err := client.Containers(ctx)
 		if err != nil {
 			return err
 		}
 
 		var containers []*native.Container
-
-		for _, container := range filteredContainers {
+		for _, container := range allContainers {
 			nativeContainer, err := containerinspector.Inspect(ctx, container)
 			if err != nil {
 				continue
 			}
 			if nativeContainer.Process == nil || nativeContainer.Process.Status.Status != containerd.Running {
 				continue
 			}
-			containers = append(containers, nativeContainer)
+			info, err := container.Info(ctx)
+			if err != nil {
+				return err
+			}
+			networkLabels, ok := info.Labels[labels.Networks]
+			if !ok {
+				continue
+			}
+			var networks []string
+			if err := json.Unmarshal([]byte(networkLabels), &networks); err != nil {
+				return err
+			}
+			for i := range len(networks) {
```

Comment:
```suggestion
			for _, network := range networks
```
for better readability ;)

Reply:
Thanks!!!